src/web/components/MostViewed/MostViewedFooter/MostViewedFooter.tsx

```diff
@@ -81,95 +78,38 @@ const mostPopularAdStyle = css`
     ${labelStyles};
 `;
 
-const secondTierStyles = css`
-    border-left: 1px solid ${border.secondary};
-    border-right: 1px solid ${border.secondary};
-
-    ${from.tablet} {
-        padding-top: 24px;
-    }
-`;
-
 interface Props {
     sectionName?: string;
     pillar: Pillar;
     ajaxUrl: string;
 }
 
-function buildSectionUrl(ajaxUrl: string, sectionName?: string) {
-    const sectionsWithoutPopular = ['info', 'global'];
-    const hasSection =
-        sectionName && !sectionsWithoutPopular.includes(sectionName);
-    const endpoint: string = `/most-read${
-        hasSection ? `/${sectionName}` : ''
-    }.json`;
-    return joinUrl([ajaxUrl, `${endpoint}?dcr=true`]);
-}
-
-export const MostViewedFooter = ({ sectionName, pillar, ajaxUrl }: Props) => {
-    const url = buildSectionUrl(ajaxUrl, sectionName);
-    const { data, error } = useApi<
-        MostViewedFooterPayloadType | TrailTabType[]
-    >(url);
-
-    if (error) {
-        window.guardian.modules.sentry.reportError(error, 'most-viewed-footer');
-        return null;
-    }
-
-    if (data) {
-        return (
-            <div className={`content-footer ${cx(adSlotUnspecifiedWidth)}`}>
+export const MostViewedFooter = ({ sectionName, pillar, ajaxUrl }: Props) => (
+    <div className={`content-footer ${cx(adSlotUnspecifiedWidth)}`}>
+        <div
+            className={cx(stackBelow('leftCol'), mostPopularAdStyle)}
+            data-link-name="most-popular"
+            data-component="most-popular"
+        >
+            <section className={asideWidth}>
+                <h2 className={headingStyles}>Most popular</h2>
+            </section>
+            <section className={stackBelow('desktop')}>
+                <Lazy margin={300}>
+                    <MostViewedFooterData
+                        sectionName={sectionName}
+                        pillar={pillar}
+                        ajaxUrl={ajaxUrl}
+                    />
+                </Lazy>
                 <div
-                    className={cx(stackBelow('leftCol'), mostPopularAdStyle)}
-                    data-link-name="most-popular"
-                    data-component="most-popular"
+                    className={css`
+                        margin: 0.375rem 0 0 0.625rem;
```

Comment:
i would prefer `px` rather than `rem`

Reply:
Me too. Done.

Sadly, the design system's `space`s didn't fit here